keyboards/ergo42/keymaps/default-underglow/rules.mk

```diff
@@ -0,0 +1,5 @@
+ifndef QUANTUM_DIR
```

Comment:
This block isn't needed, you should only need the `RGBLIGHT_ENABLE = yes` line in this file.

The rest is a holdover from the old makefile system, that just hasn't been cleaned up yet, and isn't necessary